strawberryfields/parameters.py

```diff
@@ -168,6 +168,9 @@ def par_evaluate(params):
 
     Args:
         params (Sequence[Any]): parameters to evaluate
+        dtype (None, np.dtype): NumPy data type to optionally cast atomic symbols
```

Comment:
To be more accurate, should bracketed part include `tf.dtype` (or equivalent)?

Reply:
Good point, done.